### PR TITLE
Works and About Myself Branch

### PR DESCRIPTION
**Pull request by rodrigopenayo98**
A new section is developed in the visual area of the mobile application. (CSS File)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -83,3 +98,236 @@ body {
...
   margin-right: 15px;
 }
+
+#section2 {
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  padding-top: 114px;
+  align-items: center;
+}
```
Comment (PinkMoon25) [inline styles.css:108]: - [ ] Good work so far, however as mentioned in the requirements, we have to use `grid` for the layout of the whole section, and `flexbox` for the layout of each individual project card, currently you have used flex for the layout of the whole section and grid for the u elements, using grid and flex will give you a lot of freedom in responsive design and both are crucial skills to master, please fix this
image

File: index.html [hunk 2]
```diff
@@ -36,35 +33,191 @@ <h1>
...
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>
```
Comment (PinkMoon25) [inline index.html:86]: - [ ] Please Implement the button interactions for all these buttons on the page 
image

Figma design 
image

File: styles.css [hunk 3]
```diff
@@ -83,3 +98,236 @@ body {
...
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 50px;
+  line-height: 52px;
+  margin-bottom: 20px;
+  margin-top: 114px;
+}
```
Comment (PinkMoon25) [inline styles.css:269]: - [ ] According to the figma the font size of About Myself should be `40px`, currently it is `50px` 
- [ ] [OPTIONAL] Also I would recommend removing the `<br>` tag you have used as doing so will make sure our design matches the figma design

File: index.html [hunk 4]
```diff
@@ -36,35 +33,191 @@ <h1>
...
+          </li>
+        </ul>
+        <div class="categories">
+          <h4>Framekors</h4>
+          <img src="./image/Union.png"/>  
+        </div>
+        <div class="categories">
+          <h4>Skills</h4>
+          <img src="./image/Union.png"/>
+        </div>
```
Comment (PinkMoon25) [inline index.html:220]: - [ ] For the Languages section we have to make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`, like :point_down: 
```
 <ul>
      <li> Language</li>
        <ul>
          <li>Javascript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>Css</li>
        </ul>
```

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @rodrigopenayo98 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
follows figma template :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @rodrigopenayo98 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Proper linter check ✅ 
- Changes recommended followed  👍

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.



Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**